css-layout-classes/styles.css

```diff
@@ -0,0 +1,53 @@
+#container {
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds {
+  flex-basis: 66.6%;
```

Comment:
This is totally fine but it's better if we could rely on `calc` so it does pixel perfect calculations